src/v/pandaproxy/handlers.cc

```diff
@@ -215,10 +215,39 @@ subscribe_consumer(server::request_t rq, server::reply_t rp) {
 
     return rq.ctx.client
       .subscribe_consumer(group_id, member_id, std::move(req_data.topics))
-      .then([group_id, rp{std::move(rp)}]() mutable {
+      .then([rp{std::move(rp)}]() mutable {
           // nothing to do!
           return std::move(rp);
       });
 }
 
+ss::future<server::reply_t>
+consumer_fetch(server::request_t rq, server::reply_t rp) {
+    auto fmt = parse_serialization_format(rq.req->get_header("Accept"));
+    if (fmt == ppj::serialization_format::unsupported) {
```

Comment:
should this be a function `ppj::is_accept_format_supported(fmt)` and it be an array? or we only intent to support json

Reply:
We intend to support other formats in the future. The result is used to instantiate a stateful deserialiser that constructs other deserialisers, that eventually dispatch the work based on the format.